docs/src/components/about/AboutHero.tsx

```diff
@@ -0,0 +1,178 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import { styled, keyframes } from '@mui/material/styles';
+import Section from 'docs/src/layouts/Section';
+import GradientText from 'docs/src/components/typography/GradientText';
+import TeamStatistics from 'docs/src/components/about/TeamStatistics';
+
+const teamPhotos = [
+  {
+    img: '/static/blog/2023-chamonix-retreat/skiers.jpeg',
+    title: 'MUI team members standing lined-up in the snow with their skigear.',
+  },
+  {
+    img: '/static/blog/2022-tenerife-retreat/group-photo.jpeg',
+    title: 'Photo of the MUI team in front of the pool at our accommodations in Tenerife',
+  },
+  {
+    img: '/static/blog/2023-chamonix-retreat/team-dinner.jpeg',
+    title: 'Members of the MUI team sitting around a large wooden dining table.',
+  },
+  {
+    img: '/static/about/working-table-tenerife.jpeg',
+    title: 'The Toolpad team working together on a heads-down moment in Tenerife.',
+  },
+  {
+    img: '/static/blog/2022-tenerife-retreat/scuba-gear.jpeg',
+    title:
+      'MUI team members and their diving instructors pose in scuba gear before a successful scuba diving lesson.',
+  },
+  {
+    img: '/static/blog/2022-tenerife-retreat/outdoor-focus-group.jpeg',
+    title:
+      'An impromptu focus group gathered next to the pool with laptops to discuss cross-team marketing strategies.',
+  },
+  {
+    img: '/static/about/working-table-portugal.jpeg',
+    title: 'MUI team members working together on a heads-down moment in Portugal.',
+  },
+  {
+    img: '/static/about/snow-tea.jpeg',
+    title: 'The team shares a cup of tea up in the mountains of Chamonix, France.',
+  },
+  {
+    img: '/static/about/portugal-sight-seeing.jpeg',
+    title: 'MUI team selfie while sightseeing in Lisbon, Portugal.',
+  },
+];
+
+const ImageContainer = styled('div')(() => ({
+  display: 'flex',
+  gap: 16,
+  justifyContent: 'center',
+}));
+
+const Image = styled('img')(({ theme }) => ({
+  width: 400,
+  height: 300,
```

Comment:
Maybe to make the `<img>` equal to these dimensions?

```suggestion
  width: 400,
  height: 300,
  boxSizing: 'content-box',
```